distutils/dist.py

```diff
@@ -1125,20 +1124,32 @@ def write_pkg_file(self, file):
                 self.classifiers or self.download_url):
             version = '1.1'
 
+        # required fields
         file.write('Metadata-Version: %s\n' % version)
         file.write('Name: %s\n' % self.get_name())
         file.write('Version: %s\n' % self.get_version())
-        file.write('Summary: %s\n' % self.get_description())
-        file.write('Home-page: %s\n' % self.get_url())
-        file.write('Author: %s\n' % self.get_contact())
-        file.write('Author-email: %s\n' % self.get_contact_email())
-        file.write('License: %s\n' % self.get_license())
+
+        # optional fields
+        summary = self.get_description()
+        if summary:
+            file.write('Summary: %s\n' % summary)
```

Comment:
I'd like to see this pattern replaced by a single function call that fits on one line, something like `maybe_write(file, 'Summary', self.get_description())`